src/app/shared/services/record.service.ts

```diff
@@ -31,29 +31,35 @@ import { AppConfig } from '../../app.config';
 export class RecordService {
```

Comment:
Now that this service does everything maybe better to rename it to `ApiCallService` or `ApiService` or `ApiClientService` or something similar.